e2e/docker-compose.cypress.dev.yml

```diff
@@ -2,7 +2,7 @@ version: '2.1'
 
 services:
   cypress:
-    image: cypress/included:12.17.0
+    image: cypress/included:13
```

Comment:
do we need both compose files?

Reply:
It says [here in readme](https://github.com/inspirehep/inspirehep/blob/e869894d75c61878112e71277156409cf70b40de/README.md#:~:text=To%20enable%20mounting%20backend%20code%20and%20live%20update%2C%20just%20use%20e2e/docker%2Dcompose.cypress.dev.yml%20instead.) that it was created with a purpose, but tbh I never used it for anything so I can delete

Reply:
ok, let's also cleanup the README file please